src/react/stylesheets/_colors.scss

```diff
@@ -1,31 +1,4 @@
-// Shades of Blue
-$cc-color-air-force-blue: #61889e;
-$cc-color-alice-blue: #f2f9fe;
-$cc-color-berkeley-blue: #003262;
-$cc-color-blue-lagoon: #065a7d;
-$cc-color-curious-blue: #2683bc;
-$cc-color-lochmara: #1f7099;
-
-// Shades of Blue/Gray
-$cc-color-bali-hai: #8098a5;
-
-// Shades of Gray
-$cc-color-black: #000;
-$cc-color-night-rider: #333;
-$cc-color-dim-gray: #666;
-$cc-color-nobel: #999;
-$cc-color-whisper: #eee;
-$cc-color-white: #fff;
-$cc-color-white-smoke: #f6f6f6;
-$cc-color-very-light-grey: #ccc;
-
-// Shades of Red
-$cc-color-dark-tangerine: #f1a91e;
-$cc-color-harley-davidson-orange: #cf1715;
-$cc-color-venetian-red: #c10006;
-
-// Shades of Green
-$cc-color-salem: #0d863d;
+@import './legacy_colors';
```

Comment:
@developish - Those aren't "legacy" colors. We moved to a fixed color palette to be used throughout the app. It seems like the note, that these color names come from some convention we established, was dropped.

Reply:
We should move these notes [about the color naming convention](https://github.com/sis-berkeley-edu/calcentral/blob/master/src/assets/stylesheets/colors.scss#L4)